lib/matplotlib/tests/test_image.py

```diff
@@ -402,6 +415,7 @@ def test_bbox_image_inverted():
 
     bbox_im = BboxImage(Bbox([[100, 100], [0, 0]]))
     bbox_im.set_data(image)
+    bbox_im.set_clip_on(False)
     axes.add_artist(bbox_im)
```

Comment:
Yes.  I plan to replace this with a proper `BboxImage` test.  The fun of this is that `BboxImage` isn't even terribly clearly defined as to what it's supposed to do in the first place...  I'll probably use some of the related examples as a starting point.